demisto_sdk/commands/init/contribution_converter.py

```diff
@@ -249,13 +250,17 @@ def generate_readme_for_pack_content_item(self, yml_path: str) -> None:
         """
         file_type = find_type(yml_path)
         file_type = file_type.value if file_type else file_type
+        dir_output = os.path.dirname(os.path.realpath(yml_path))
```

Comment:
move these 2 lines to be with the the line after the condition so or move that one up, just so these 3 will be together